### PR TITLE
feat: 🎸 [JIRA:IOSSDKBUG-434] SortFilterFeedbackBar enhancement list height

### PR DESCRIPTION
**Pull request by restaurantt**
1. search bar in list can be optionally shown.
2. list mode, adjust popover height for 3 situations, less than 50% screen height, medium, large.
Simulator Screenshot - iPhone 16 Plus - 2024-11-01 at 15 37 47
Simulator Screenshot - iPhone 16 Plus - 2024-11-01 at 15 37 50
Simulator Screenshot - iPhone 16 Plus - 2024-11-01 at 15 37 52

### CODE REVIEW

File: Sources/FioriSwiftUICore/DataTypes/SortFilter+DataType.swift [hunk 1]
```diff
@@ -363,6 +363,8 @@ public extension SortFilterItem {
         /// itemLayout is used when listPickerMode is filterFormCell, otherwise is ignored.
         public var itemLayout: OptionListPickerItemLayoutType = .fixed
         public var displayMode: DisplayMode = .automatic
+        /// If seachBar in list display mode is shown. Default is `false`.
```
Comment (dyongxu) [inline Sources/FioriSwiftUICore/DataTypes/SortFilter+DataType.swift:366]:
```suggestion
        /// If seachBar in lister picker is shown. Default is `false`.
```

**Review by dyongxu — APPROVED** [review]

File: Sources/FioriSwiftUICore/DataTypes/SortFilter+DataType.swift (hunk 1 above)
Comment (janhuachu) [reply Sources/FioriSwiftUICore/DataTypes/SortFilter+DataType.swift]: Should be "list picker" not "lister picker"

File: Sources/FioriSwiftUICore/DataTypes/SortFilter+DataType.swift [hunk 2]
```diff
@@ -363,6 +363,8 @@ public extension SortFilterItem {
         /// itemLayout is used when listPickerMode is filterFormCell, otherwise is ignored.
         public var itemLayout: OptionListPickerItemLayoutType = .fixed
         public var displayMode: DisplayMode = .automatic
+        /// If seachBar in lister picker is shown. Default is `false`.
+        public var isSearchBarHidden: Bool = false
```
Comment (janhuachu) [inline Sources/FioriSwiftUICore/DataTypes/SortFilter+DataType.swift:367]: line 366 "lister picker" should be "list picker"

File: Sources/FioriSwiftUICore/DataTypes/SortFilter+DataType.swift [hunk 3]
```diff
@@ -363,6 +363,8 @@ public extension SortFilterItem {
         /// itemLayout is used when listPickerMode is filterFormCell, otherwise is ignored.
         public var itemLayout: OptionListPickerItemLayoutType = .fixed
         public var displayMode: DisplayMode = .automatic
+        /// If seachBar in lister picker is shown. Default is `false`.
```
Comment (dyongxu) [inline Sources/FioriSwiftUICore/DataTypes/SortFilter+DataType.swift:366]:
```suggestion
        /// If seachBar in list picker is shown. Default is `false`.
```

File: Sources/FioriSwiftUICore/DataTypes/SortFilter+DataType.swift (hunk 2 above)
Comment (dyongxu) [reply Sources/FioriSwiftUICore/DataTypes/SortFilter+DataType.swift:367]: updated.

**Review by billzhou0223 — APPROVED** [review]

File: Sources/FioriSwiftUICore/DataTypes/SortFilter+DataType.swift [hunk 4]
```diff
@@ -363,6 +363,8 @@ public extension SortFilterItem {
         /// itemLayout is used when listPickerMode is filterFormCell, otherwise is ignored.
         public var itemLayout: OptionListPickerItemLayoutType = .fixed
         public var displayMode: DisplayMode = .automatic
+        /// If seachBar in list picker is shown. Default is `false`.
```
Comment (dyongxu) [inline Sources/FioriSwiftUICore/DataTypes/SortFilter+DataType.swift:366]:
```suggestion
        /// If searchBar in list picker is shown. Default is `false`.
```

**Pull request merged**